djangosaml2idp/views.py

```diff
@@ -184,10 +184,10 @@ def build_authn_response(self, user, authn, resp_args, processor: BaseProcessor,
         )
         return authn_resp
 
-    def render_login_html_to_string(self, template_name, context=None, request=None, using=None):
+    def render_login_html_to_string(self, context=None, request=None, using=None):
         """ Render the html response for the login action. Can be using a custom html template if set on the view. """
         default_login_template_name = 'djangosaml2idp/login.html'
-        custom_login_template_name = getattr(self, 'login_html_template')
+        custom_login_template_name = getattr(self, 'login_html_template', None)
         if custom_login_template_name:
```

Comment:
The only other call to `render_login_html_to_string` is from `create_html_response` and it only passes two kwargs (context and request), no args. So I think this is a leftover of a refactor.

In the code template_name is not being used, so I just removed it and handled the case there there is no `login_html_template` set.